src/objective-c/RxLibrary/GRXImmediateSingleWriter.m

```diff
@@ -75,7 +75,11 @@ - (void)setState:(GRXWriterState)newState {
   return;
 }
 
+// Overrides [requestWriter(Transformations):map:] for Protocol Buffers
+// encoding.
```

Comment:
Add the explanation that we need this to return a `GRXImmediateSingleWriter`, and the method we're overriding wouldn't.